pymatviz/ptable.py

```diff
@@ -299,21 +299,21 @@ def elem_colors(self) -> dict[str, ColorType]:
     @elem_colors.setter
     def elem_colors(
         self,
-        elem_colors: Literal["vesta", "jmol"] | dict[str, ColorType] = "vesta",
+        elem_colors: ElemColors | dict[str, ColorType] | None = ElemColors.vesta,
```

Comment:
Wondering why do we need a `None` as default here? which would resolve to `vesta` anyway?

Reply:
i imagine someone who wants to set the colors back to default and doesn't know what the default is might just try

```py
projector.elem_colors = None
```

and see what happens. maybe that's just me in which i'm open to changing this

Reply:
> i imagine someone who wants to set the colors back to default and doesn't know what the default is might just try
> 
> ```python
> projector.elem_colors = None
> ```
> 
> and see what happens.

I think if a user don't read the docstring/manual at all, they may not image setting `None` would set the colors to default in the first place (they might expect the default to be a `dict` mapping). On the other hand, if they read the docstring, they would know how to set the colors, so I think `None` adds to ambiguity here :)

Reply:
fair, feel free to remove the `None`

Reply:
Okay!